spec/models/kalibro_range_spec.rb

```diff
@@ -16,6 +17,32 @@
     it { is_expected.to validate_numericality_of(:end) }
     it { is_expected.to validate_uniqueness_of(:beginning).
          scoped_to(:metric_configuration_id).with_message("Should be unique within a Metric Configuration") }
+
+    context 'with invalid beginning or end' do
+      subject { FactoryGirl.build(:kalibro_range_with_id) }
+
+      before :each do
+        # Make sure the uniqueness in the Metric Configuration scope isn't triggered, since
+        # we only want to test validations that deal exclusively with the range values
+        ActiveRecord::Validations::UniquenessValidator.any_instance.stubs(:validate_each)
```

Comment:
This seems awkward to me.
If the stub was not there, would the uniqueness validator be called? If so, it would always pass on the validation, right? We don't have (or should not have) any ranges saved on the database.

Is this really necessary?